scib/metrics/ari.py

```diff
@@ -7,15 +7,16 @@
 
 
 def ari(adata, group1, group2, implementation=None):
-    """ Adjusted Rand Index
+    """
+    Adjusted Rand Index.
     The function is symmetric, so group1 and group2 can be switched
-    For single cell integration evaluation the scenario is:
-        predicted cluster assignments vs. ground-truth (e.g. cell type) assignments
+    For single cell integration evaluation the comparison is between predicted cluster
+    assignments and the ground truth (e.g. cell type)
 
     :param adata: anndata object
     :param group1: string of column in adata.obs containing labels
     :param group2: string of column in adata.obs containing labels
```

Comment:
I know this is symmetric so it doesn't matter but I would give these clearer names (and make them consistent across functions).

Reply:
agreed

Reply:
I think this should be cluster and label for ARI, and not "batch".

Reply:
I'll keep this for now and deal with it in another PR